### DISCUSSION TITLE
Please allow the user to change the title of a gist instead of forcing him to create a dummy file just for this

### DISCUSSION THREAD

**Discussion by adriancuadrado** [community/community#86555] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

According to [this StackOverflow post](https://stackoverflow.com/a/62398008/5744858), there is no way to change the title of a gist other than creating a dummy text file that starts with a space or a special character that takes precedence in the ascii table. It doesn't make sense that people have to resort to this kind of hack for something as simple as changing the title of a gist.

It would also be awesome if we could manually change the order in which files appear instead of having to prefix each and every one of them with a number or something like someone said in [this StackOverflow post](https://stackoverflow.com/a/37745005/5744858).

**Comment by ernstki** [comment]
Duplicate of https://github.com/orgs/community/discussions/13772.

Having the first file alphabetically show up as the title of the gist (and in search results) is pretty suboptimal.

I've lost track of how many times I have to rename files to `zz_something` to work around this. Or do without a `.gitignore` because if I create one in a Gist, my Gist will be named `.gitignore`.

One possible solution might be to add a checkbox designating the "primary" file for the gist.
